design/FY2019/AddCoilandWaterheaterChoicesforDesuperheater.md

```diff
@@ -0,0 +1,75 @@
+# Add More Coil/WaterHeater Choices for Coil:WaterHeating:Desuperheater
+
+
+**Yueyue Zhou**
+
+**National Renewable Energy Laboratory**
+
+**April 2, 2019**
+ 
+
+## Justification for New Feature ##
+
+Desuperheater is a widely used new technology making use of waste heat from HVAC system to heat domestic hot water. Desuperheater is available in E+ only supporting 
+a very limited amount of HVAC systems and water heater tanks, while the most widely used occasion for desuperheater: ground source heat pump system is not yet available.
+ Given only 8 valid choices of cooling and refrigeration components and only 1 tank option, it constrained the variations people modeling their hybrid systems and impeded 
+ the promotions of using this new energy efficiency techonology. Users and interface developers are requesting EnergyPlus broaden its capability of modeling desuperheater 
+ with more coils and tanks options.Some other energy modeling  software tools like Rem/Rate, Ekotrope, Wrightsoft are already capable to model this hybrid GSHP system with 
+ simple assumptions and methods. Since EnergyPlus already supports some systems, it would be more straight-forward to add more enumarations to EnergyPlus.
+
+
+## Overview ##
+
+EnergyPlus would be modified to add more enumarations in Coil:WaterHeating:Desuperheater object's heating source object type and tank object type fields.
+
+## Approach ##
+
+EnergyPlus already handles HVAC/water heating hybrid systems for several coil types. The workflow is straight-forward to add the same structure as those systems. The steps to new enumarations are given below.
+
+1. IDD file modification: **Coil:WaterHeating:Desuperheater** A7 add **WaterHeater:Stratified** ; A9 add **Coil:Cooling:WaterToAirHeatPump:EquationFit**, 
+**Coil:Cooling:WaterToAirHeatPump:ParameterEstimation**, **Coil:Cooling:DX:MultiSpeed** , etc. 
```

Comment:
The only comment would be to list the entire set of objects to be included as source objects. This is the current list.

    A5 , \field Heating Source Object Type
       \required-field
       \type choice
       \key Coil:Cooling:DX:SingleSpeed
       \key Coil:Cooling:DX:VariableSpeed
       \key Coil:Cooling:DX:TwoSpeed
       \key Coil:Cooling:DX:TwoStageWithHumidityControlMode
       \key Refrigeration:CompressorRack
       \key Refrigeration:Condenser:AirCooled
       \key Refrigeration:Condenser:EvaporativeCooled
       \key Refrigeration:Condenser:WaterCooled

Reply:
Also, WaterHeater:Stratified does not have a compressor so could not be a source for waste heat.

Reply:
My comment that water tank does not have a compressor makes me wonder why Refrigeration:Condenser* was included in this list since the waste heat is generated by the Refrigeration:CompressorRack? I guess you could use one or the other?

Reply:
Hi, Richard, thanks for your comment. The object you've found might be the “Coil:Heating:Desuperheater”, while the modified one is “Coil:WaterHeating:Desuperheater”, and the corresponding fields are A7 and A9. This coil makes use of HVAC waste heat to heat domestic hot water. Therefore, a stratified water tank can be added as a new tank object type since the only one available now is the mixed water tank. I will add the entire set of IDD object list to the NFP. Thanks for your suggestion.

Reply:
Of course. I searched for :Desuperheater, and that's the first one I came to. Comment still applies. In the NFP, list the new objects for A7 and A9 in the object you intend to modify so we can see which objects are being added. Also, is it just Coil:WaterHeating:Desuperheater being modified when Coil:Heating:Desuperheater could also be modified with similar changes (e.g., Coil:Cooling:DX:Multispeed is not listed in Coil:Heating:Desuperheater field A5)? Seems simple enough to correct both desuperheaters?

Reply:
Richard, I've updated the NFP to include the entire list of A7 and A9. Hope it would be more clear. I agree that extending the Coil:Heating:Desuperheater object might be similar in some way, we would consider it after finishing the water heater desuperheater part. Thanks.